modules/consul-cluster/main.tf

```diff
@@ -28,17 +28,19 @@ resource "aws_autoscaling_group" "autoscaling_group" {
   health_check_grace_period = "${var.health_check_grace_period}"
   wait_for_capacity_timeout = "${var.wait_for_capacity_timeout}"
 
-  tag {
-    key                 = "Name"
-    value               = "${var.cluster_name}"
-    propagate_at_launch = true
-  }
-
-  tag {
-    key                 = "${var.cluster_tag_key}"
-    value               = "${var.cluster_tag_value}"
-    propagate_at_launch = true
-  }
+  tags = [
+    {
+      key                 = "Name"
+      value               = "${var.cluster_name}"
+      propagate_at_launch = true
+    },
+    {
+      key                 = "${var.cluster_tag_key}"
+      value               = "${var.cluster_tag_value}"
+      propagate_at_launch = true
+    },
+    "${var.tags}",
```

Comment:
Does this actually work? I'd expect you to have to call `concat`.

Reply:
Yes it does, I've tested this by pulling my fork of the module to an existing infrastructure configuration and the tags got defined correctly. I considered using `concat` but found this construction to read better.

Reply:
Ah, neat, thx!